### DISCUSSION TITLE
titles in tp.system.suggester??

### DISCUSSION THREAD

**Discussion by Cryptodarktrader** [SilentVoid13/Templater#1226] [Ideas & Feature Requests]
I think that at the moment it is not possible, or I do not know how to do it, that is why I would like to suggest incorporating the possibility of establishing a title when one tries to put pop-up windows with options, since I currently have templates where when the note is created you It asks several things and sometimes just with the list of answers it can create confusion without a question that tells you what is being chosen.

**Comment by Zachatoo** [comment]
You can set a placeholder to tell you what the suggester is for.

Docs [here](https://silentvoid13.github.io/Templater/internal-functions/internal-modules/system-module.html#tpsystemsuggestertext_items-string--item-t--string-items-t-throw_on_cancel-boolean--false-placeholder-string---limit-number--undefined).

```js
<% await tp.system.suggester(["Yes", "No"], ["true", "false"], false, "Title here!") %>
```

**Comment by Cryptodarktrader** [comment]
wow It works perfectly, I don't know why I didn't understand how to put it. I thought it couldn't be done, thank you very much!!!
